kraken/pvc/pvc_scenario.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import yaml
+import logging
+import time
+import kraken.cerberus.setup as cerberus
+import kraken.kubernetes.client as kubecli
+
+# Reads the scenario config and creates a temp file to fill up the PVC
+def run(scenarios_list, config):
+    failed_post_scenarios = ""
+    for app_config in scenarios_list:
+        if len(app_config) > 1:
+            with open(app_config, "r") as f:
+                config_yaml = yaml.full_load(f)
+                scenario_config = config_yaml["pvc_scenario"]
+                pod_name = scenario_config.get("pod_name", "")
+                namespace = scenario_config.get("namespace", "")
```

Comment:
Maybe exit if namespace/podname is empty? Thoughts?

Reply:
good idea, it's done